dashboards-observability/common/constants/explorer.ts

```diff
@@ -233,6 +234,28 @@ export const DEFAULT_STATS_CHART_PARAMETERS: DefaultStatsChartParametersProps =
   },
   DefaultTextColor: '#FFFFFF',
 };
+export interface DefaultBarChartStylesProps {
+  BarMode: string;
+  GroupWidth: number;
+  BarWidth: number;
+  LineWidth: number;
+  LabelSize: number;
+}
+
+export const DEFAULT_BAR_CHART_STYLES: DefaultBarChartStylesProps = {
+  BarMode: 'group',
+  GroupWidth: 0.7,
+  BarWidth: 0.97,
+  LineWidth: 1,
+  LabelSize: 12,
+};
+
+export const SIMILAR_VIZ_TYPES = [
```

Comment:
Check if this object is needed now, else remove this.

Reply:
yes it is used in bar.tsx as default value if not applied in chart styles